101-ubuntu-mate-desktop-vscode/README.md

```diff
@@ -0,0 +1,14 @@
+# Deploy an Ubuntu Mate Desktop VM with VSCode
```

Comment:
love this sample - can you add a bullet in the readme here about how to connect to the VM via GUI?

Reply:
Done.